src/viz/drawing.jl

```diff
@@ -0,0 +1,281 @@
+        with_edge_labels::Bool=true,
+        with_node_labels::Bool=true,
+        label_alpha::Float64=.35
+        )
+
+ Draw a hypergraph `h` as an Euler diagram, using the library [HyperNetX](https://github.com/pnnl/HyperNetX).
+
+ **Arguments**
+
+ * `h` : the hypergraph to draw
+ * `type` : how the hypergraph will be drawn. If `type=HyperNetX`, the hypergraph will be represented as a Euler Diagram
+ * `width` : width of the figure
+ * `height` : height of the figure
+ * `node_labels` : node labels to be shown
+ * `edge_labels` : edge labels to be shown
+ * `collapse_nodes` : draws the hypergraph gotten by identifying nodes contained by the same edges (from HyperNetX)
+ * `collapse_edges` : draws the hypergraph gotten by identifying edges containing the same nodes (from HyperNetX)
+ * `no_border` : indicates wheter the figure should have a border
+
+For more details about the other parameters, please refer to the library [HyperNetX](https://github.com/pnnl/HyperNetX).
+"""
+function draw(
+        h::Hypergraph,
+        type::Type{HyperNetX};
+        width::Int=10,
+        height::Int=10,
+        node_labels::Union{Dict{Int, String}, Nothing}=nothing,
+        edge_labels::Union{Dict{Int, String}, Nothing}=nothing,
+        collapse_nodes::Bool=false,
+        collapse_edges::Bool=false,
+        pos::Union{Dict{Int,Pair{Int,Int}}, Nothing}=nothing,
+        with_color::Bool=true,
+        with_node_counts::Bool=false,
+        with_edge_counts::Bool=false,
+        layout::PyObject=nx.spring_layout,
+        layout_kwargs::Dict=Dict{String, Any}(),
+        ax::Union{PyObject, Nothing}=nothing,
+        no_border::Bool=false,
+        edges_kwargs::Dict=Dict{String, Any}(),
+        nodes_kwargs::Dict=Dict{String, Any}(),
+        edge_labels_kwargs::Dict=Dict{String, Any}(),
+        node_labels_kwargs::Dict=Dict{String, Any}(),
+        with_edge_labels::Bool=true,
+        with_node_labels::Bool=true,
+        label_alpha::Float64=.35
+        )
+
+    h_hnx = _convert_to_hnx(h;
+        node_labels=node_labels,
+        edge_labels=edge_labels
+        )
+
+    if collapse_nodes
+        h_hnx = h_hnx.collapse_nodes()
+    end
+    if collapse_edges
+        h_hnx = h_hnx.collapse_edges()
+    end
+
+    if isnothing(ax)
+        fig = plt.figure(figsize=[width,height])
+        ax = plt.gca()
+
+        if no_border
+            ax.axis("off")
+        end
+    end
+
+    hnx.draw(h_hnx,
+        pos=pos,
+        with_color=with_color,
+        with_node_counts=with_node_counts,
+        with_edge_counts=with_edge_counts,
+        layout=layout,
+        layout_kwargs=layout_kwargs,
+        ax=ax,
+        edges_kwargs=edges_kwargs,
+        nodes_kwargs=nodes_kwargs,
+        edge_labels_kwargs=edge_labels_kwargs,
+        node_labels_kwargs=node_labels_kwargs,
+        with_edge_labels=with_edge_labels,
+        with_node_labels=with_node_labels,
+        label_alpha=label_alpha
+    )
+end
+
+
+"""
```

Comment:
Missing tests for this function

Reply:
The function `_convert_to_hnx()` in `drawing.jl` is currently not exported by `SimpleHypergraphs.jl`. How can I test - in a neat way - private functions?

Reply:
You can use a non-exported function simply prepending it with module name that is 
`SimpleHypergraphs._convert_to_hnx()` should work